test/test_components.py

```diff
@@ -1449,6 +1449,16 @@ def test_postprocess(self):
         result_ = component.postprocess({"text": text, "entities": entities})
         assert result == result_
 
+        # Test split entity is merged
+        text = "Wolfgang lives in Berlin"
```

Comment:
Thanks for the PR @payoto !

This test is no longer passing with your change. Can you fix?

Reply:
Yes I can, FYI the fix I can see would be to edit the reference result which includes some of the `('', None)`, is that an ok fix?

Entries are still only merged when the user passes the specific argument.

This change does include a change in the behaviour of the postprocess method. An alternative way to fix this bug which would not change the default behavior would be to ignore entries which are `('', None)` when merging neighbors. Which solution do you prefer?

Personally I think the first one is the way to go but you certainly have more context as to how these features might be used than I do.

Reply:
Ok in the end I went with the change that creates least changes of behaviour, so I do the merge by not considering the entries which say `('', None)` in the merge strategy